.github/workflows/go-test.yaml

```
@@ -35,6 +35,8 @@ jobs:
       - name: Test plugin with ceph commands
```

Can we get these tests all under different `name`s in the test? Or is that going to be a separate PR?
```suggestion
      - name: rook-ceph health
```